Src/PerformanceCollector/Shared/Implementation/QuickPulse/Service contract/TelemetryDocument/EventTelemetryDocument.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    internal struct EventTelemetryDocument : ITelemetryDocument
+    {
+        [DataMember(EmitDefaultValue = false)]
+        public Guid Id { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string Version { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public DateTimeOffset Timestamp { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string OperationId { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string Name { get; set; }
+        
+        [DataMember(EmitDefaultValue = false)]
+        public KeyValuePair<string, string>[] Properties { get; set; }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string DocumentType
+        {
+            get
+            {
+                return TelemetryDocumentType.Event.ToString();
+            }
+
+            private set
+            {
+            }
+        }
+
+        [DataMember(EmitDefaultValue = false)]
+        public string[] DocumentStreamIds { get; set; }
```

Comment:
This is different. _DocumentStreamId_ doesn't have anything to do with the telemetry item in broader SDK sense, but rather with the QuickPulse's concept of the document stream itself. This value is used to identify which document streams this telemetry item belongs to. We can't afford to send out the entire document 10 times for 10 different UXs, so we must maintain a list of 10 stream ids here in that case.